Mathlib/RingTheory/Localization/Integral.lean

```diff
@@ -466,3 +466,35 @@ lemma isAlgebraic_of_isFractionRing {R S} (K L) [CommRing R] [CommRing S] [Field
     apply IsIntegral.tower_top (R := R)
     apply IsIntegral.map (IsScalarTower.toAlgHom R S L)
     exact Algebra.IsIntegral.isIntegral (s : S)
+
+namespace Polynomial
+
+lemma isLocalization {R} [CommRing R] (S : Submonoid R) (A) [CommRing A] [Algebra R A]
```

Comment:
I wonder if this is the correct file for this lemma, it seems to me that it has nothing to do with Integral elements. Perhaps in Localization.Algebra?

Reply:
The proof uses `IsLocalization.integerNormalization_spec`, which is proved in this file, so I think it's the right spot?

Reply:
Yes, I agree, but in a sense this is an accident: see Bourbaki, Alg. Comm, Chap III, § 2, no. 8 for the proof that localization commutes more generally with symmetric algebras (over any ring, no need for it to be commutative btw), and univariate polynomials are a special case. Now, I am of course not arguing that we should first define symmetric algebras and prove the result in that generality, it is great to have this result now. But I feel that from a math point of view this result is unrelated to integrality and should be in its "moral" home, even if it is not yet clear why it is there. Can't you simply import `isLocalization.integerNormalization_spec` in `Localization.Algebra`?